packages/ag-charts-website/src/content/docs/financial-charts/_examples/financial-charts-showcase/main.ts

```diff
@@ -10,38 +10,40 @@ const options: AgFinancialChartOptions = {
         annotations: [
             {
                 type: 'parallel-channel',
-                start: { x: { __type: 'date', value: new Date('2022-03-01').getTime() }, y: 160.15 },
-                end: { x: { __type: 'date', value: new Date('2024-06-28').getTime() }, y: 160.15 },
-                height: 12,
-                stroke: '#808080B0',
-                background: {
-                    fill: '#808080B0',
+                start: { x: { __type: 'date', value: 1698200000000 }, y: 148.0 },
```

Comment:
Nit: Specify date/time the same was as in the data, to avoid complications with different timezones.